### ISSUE TITLE
Bulk Update profiles

### ISSUE THREAD

**Issue by paulboenisch** [SC-Networks/evalanche-soap-api-connector#32]
Hello SC-Networks,

ist it possitble to bulk update/create profiles, similiar to (delete(int[] <profile-ids>): bool) with this package?

Cheers

**Comment by WorksDev** [comment]
@paulboenisch Yes this is possible, take a look at the MassUpdate method from ProfileClient.
